src/components/layout/Misc.tsx

```diff
@@ -253,6 +253,7 @@ export const TinyText: ParentComponent = (props) => {
 export const TinyButton: ParentComponent<{
     onClick: () => void;
     tag?: MutinyTagItem;
+    hidden?: boolean;
```

Comment:
I think the need for something like this is more of a symptom than a cure. let's just not have these buttons while we don't have a good way to get the correct list of them